script/testing/junit/src/dynamicTest2.java

```diff
@@ -0,0 +1,159 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.api.function.Executable;
+
+import java.io.*;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+/**
+ * one class file dynamically generate test cases for each query
+ * provide
+ */
+public class dynamicTest2 {
```

Comment:
What does this class file do? Do we still need it? Why is it called "2"?

Follow Java  class naming conventions: `DynamicTest2`